lib/connectors/kv-memory.js

```diff
@@ -201,9 +201,11 @@ KeyValueMemoryConnector.prototype.disconnect = function(callback) {
   process.nextTick(callback);
 };
 
-KeyValueMemoryConnector.prototype.flush =
+KeyValueMemoryConnector.prototype.deleteAll =
 function(modelName, options, callback) {
-  this._store = Object.create(null);
+  var modelStore = this._store[modelName];
```

Comment:
```js
var store = this._getStoreForModel(modelName);
```

See e.g. `KeyValueMemoryConnector.prototype.iterateKeys`. Feel free to use `modelStore` as the variable name though.